Jenkinsfile.github

```diff
@@ -14,24 +14,22 @@ pipeline {
         NAME = "cray-psp"
         DESCRIPTION = "Deploys resources to set up PSP in the Cray Kubernetes cluster."
         IS_STABLE = getBuildIsStable()
-        CHART_VERSION = getChartVersion(name: env.NAME, isStable: env.IS_STABLE)
     }
 
     stages {
         stage("Build") {
-            parallel {
-                stage('Chart') {
-                    steps {
-                        sh "make chart"
-                    }
-                }
+            environment {
+                CHART_VERSIONS = getChartVersion(name: "cray-psp",   chartDirectory: "charts", isStable: env.IS_STABLE)
```

Comment:
There's only 1 version here so this variable should be CHART_VERSION not CHART_VERSIONS.

Reply:
It is aligned with what's expected in Makefile, but I went ahead and changed it to `CHART_VERSION` since this repo only has one chart.